image2tensor/src/image2tensor.rs

```diff
@@ -46,6 +46,29 @@ pub fn convert_image_to_bytes(
     }
 }
 
+// standard 8bit RGB format. It should work with standard image formats such as .jpg, .png, etc
+pub fn convert_image_bytes_to_tensor_bytes(
```

Comment:
It probably makes sense to use this new function in the body of `convert_image_to_bytes`, since it seems to me to be a copy of the latter half of that one.